### PR TITLE
✨Generate operator manifests

### PR DESCRIPTION
**Pull request by alexander-demichev**
Generate operator manifests

Fixes https://github.com/kubernetes-sigs/cluster-api/issues/4271

/area operator

### CODE REVIEW

File: exp/operator/config/crd/bases/operator.cluster.x-k8s.io_bootstrapproviders.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.5.0
+  creationTimestamp: null
```
[the remark this answers is not in the document]
Comment (MarcelMue) [reply exp/operator/config/crd/bases/operator.cluster.x-k8s.io_bootstrapproviders.yaml:8]: AFAIK including these null values is normal. See e.g. here on master:
https://github.com/kubernetes-sigs/cluster-api/blob/752c727cf58bf4af94d3337c4f8e21980ee4c06a/config/crd/bases/cluster.x-k8s.io_clusters.yaml#L8

**Review by MarcelMue — APPROVED** [review]
/lgtm

**Review by vincepri — APPROVED** [review]
/approve

**Pull request merged**